infrastructure/code/code_html_test.go

```diff
@@ -108,6 +109,44 @@ func Test_Code_Html_getCodeDetailsHtml_ignored(t *testing.T) {
 	assert.Contains(t, codePanelHtml, `class="ignore-details-section"`)
 }
 
+func Test_Code_Html_getCodeDetailsHtml_ignored_customEndpoint(t *testing.T) {
+	_ = testutil.UnitTest(t)
```

Comment:
question: why not assign the returned config to a var and use it later on instead of using `config.CurrentConfig()`?